### ISSUE TITLE
Unknown File Format?

### ISSUE THREAD

**Issue by MsSamp** [photopea/photopea#3766]
One of my students has lost his project seemingly as when he went to open the psd file and even the jpg file, he recieves a message that it can't be opened and says Unknown File Format u0000..... (see attached)

We use Chromebooks and this is the first time in two years, this issue has come up.... He's not able to open any of the files related to this project. He had turned in his project to me, but had to make some edits for his final grade...he went to open it and the message appeared. I have also tried to open it and receive same message.

![VHIssue](https://user-images.githubusercontent.com/91342434/134722011-51de081a-f67e-419f-9c9c-4fc7c4503a67.png)

**Comment by photopea** [comment]
Hi, I think the file is damaged. I am sorry for this :( Are you sure, that the chromebooks where the PSD was created is updated to the latest Chrome OS?

**Comment by MsSamp** [comment]
Yes, it should be, however I will check with our IT dept to be certain. Thanks -

**Comment by photopea** [comment]
Could you try to reproduce the problem? After you save a PSD, check, if the file is zero Bytes (it should not be).

It is possible, that your browser extensions are modifying the saved file, too.

**Comment by tittlejoseph** [comment]
same with me, i had 5 of such psd, saved one after the other and i'm unable to open any one of it, just needed small modification.. really sad

**Comment by MsSamp** [comment]
Are you getting a message that it has 0 bytes? When you try to open it from within photopea?

Get Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by tittlejoseph** [comment]
no it was format error only. i don tknow what is happening, it created multiple copies on each folder and now i see all folders are empty, including the original folder. I also see in download history it says Blob type saved. i'm using e: which is memory card

**Comment by tittlejoseph** [comment]
![image](https://user-images.githubusercontent.com/11676117/135886490-db4ad8f9-2a7e-427f-8355-82697133f96b.png)

**Comment by MsSamp** [comment]
So I just submitted a ticket to have them reset your Chromebook that will make a big difference because it helped another student who had a similar situation.

If this is happening on a personal computer I have no solution to suggest because we can't fix a personal computer. It might be that there's a virus of some kind on your computer that's causing this to take place but I am not an expert I would have to direct you to see a professional.

I highly recommend if you are not using your Chromebook that you switch back to using your Chromebook. Because at least we can help you on our end even remotely he's going to go ahead and do the update anyway because I just put the ticket in. Allow for 24 hrs turnaround. If files are corrupt there's nothing we can do they just have to be deleted and the project needs to be restarted. It's an unfortunate situation it happens to all of us at some point in our lives with computers.

As for your files disappearing off of your Aspire Google Drive I have no idea why that's happening. But I can absolutely put in a ticket for that as well to have that checked out.

Get Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by MsSamp** [comment]
Another thing is you cannot open a psd or native file unless you are in a program such as Photoshop or photopea. We will be on Photoshop when we return next week it's being installed in the new computers this week so things will change very dramatically for everyone!

Get Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by tittlejoseph** [comment]
i was using Brave browser, can it be root cause?

**Comment by MsSamp** [comment]
I honestly have no idea, so you'r e using your own computer I'm guessing? I'm unfamiliar with that browser. If a file is corrupt it's usually something to do with the computer because as soon as the other student's computer was updated by our IT department he was able to save his files once again and not have them be in a weird format or with 0 bytes which means there was nothing on there no data. Give your Chromebook a try. That's the best suggestion I can give to you!

I just thought of another suggestion and that is that you can put in a ticket to the photopea help desk? Maybe they can offer help to you because again I don't know how to help someone on their personal computer. You will have to sign up for the program they use for giving help first it will direct you how to do that. You would have to click the button on top that says report an issue. In fact, they might have suggestions already from previous customers writing about similar issues. Something you can search in there is what is the preferred browser. And you're right the browser could be the issue but I honestly don't have the expertise to confirm that is the case. Good luck! And keep me posted.

Get Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by tittlejoseph** [comment]
Thanks for showing the genuine concern on fixing issue. I use windows 10 on surface pro and instead of using google chrome nowadays use browser "BRAVE", it shows no ad for YouTube and many other apps, so i like this browser.

Anyway i have recreated things from scratch using your tool again.  But this time I used Google Chrome. Not sure unable to save PSD, the button is not clicking on popup window of save. Anyway i have exported image so okay to lose the work. If unable to save after few efforts.

**Comment by tittlejoseph** [comment]
![image](https://user-images.githubusercontent.com/11676117/135902597-f60c5f1c-675e-4cf2-8bde-0c29ec3702a1.png)

**Comment by tittlejoseph** [comment]
![image](https://user-images.githubusercontent.com/11676117/135902822-a81d27e0-2f3b-4539-988b-830789c1dd96.png)

**Comment by MsSamp** [comment]
If you are unable to save a p s d that would be a reason to contact the photopea help desk and see what they can do for you. This view that you sent is 1 I'm not used to seeing and it's because you're using a different device and browser I think that they said that Safari was a good choice for photopea but that's only on Apple computers and products but I'm not sure you'd have to do a quick search on that which browsers does photopea support best or vice versa

Get Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by photopea** [comment]
Hi guys, when you press File - Save as PSD, it should create a PSD file in your device.

If it does not happen, try these steps: https://www.photopea.com/tuts/photopea-doesnt-work-what-to-do/

If you still have a problem, tell me, how exactly can I reproduce the problem. I.e. describe steps, which you have tried multiple times on your device, and they resulted in a problem every time.

If your files disappear from Google drive or from a flash drive, the problem is not on our side :( A website can not even do such things.
